data/sql/updates/pending_db_characters/rev_1659776114435282400.sql

```diff
@@ -0,0 +1,4 @@
+--
+ALTER TABLE `character_entry_point`
+ADD COLUMN `taxiPath0` INT UNSIGNED DEFAULT 0 NOT NULL AFTER `joinMapId`,
+CHANGE `taxiPath` `taxiPath1` INT UNSIGNED DEFAULT 0 NOT NULL; 
```

Comment:
This query is giving me an error, probably because some of the values for this is an empty string:
`
Truncated incorrect INTEGER value: ''
`